docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md

```diff
@@ -8,6 +8,9 @@ ms.assetid: a6e6e65c-347f-4494-9457-653bf29baac2
 
 The XML Schema Definition (Xsd.exe) tool generates XML schema or common language runtime classes from XDR, XML, and XSD files, or from classes in a runtime assembly.
 
+The XML Schema Definition tool (Xsd.exe) usually can be found in the following path:  
+C:\Program Files (x86)\Microsoft SDKs\Windows\ ***\<version\>*** \bin\NETFX ***\<version\>*** Tools
```

Comment:
Thanks for doing this change too. I missed it some how so here is the suggestion to fix it.

```suggestion
The XML Schema Definition tool (Xsd.exe) usually can be found in the following path:\
*C:\Program Files (x86)\Microsoft SDKs\Windows\{version}\bin\NETFX {version} Tools\*
```